core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.html

```diff
@@ -10,6 +10,19 @@
       </i>
     </span>
   </div>
+  <div class="story-node-field">
+    <label for="storyNodeDescription" class="form-heading">Chapter Description</label>
+    <textarea id="storyNodeDescription" type="text" class="form-control protractor-test-add-chapter-description"
+              ng-model="editableDescription" ng-blur="updateDescription(editableDescription)"
+              placeholder="Enter the description for the chapter." maxlength="<MAX_CHARS_IN_CHAPTER_DESCRIPTION>" ng-trim="false"
```

Comment:
This should be `<[MAX_CHARS_IN_CHAPTER_DESCRIPTION]>`. Make sure to test every UI changes.

Reply:
Done.